requirements.txt

```diff
@@ -4,3 +4,4 @@ protobuf
 onnx>=1.2.1
 scikit-learn>=0.19
 onnxconverter-common>=1.7.0
+scikit-learn<=1.1.1
```

Comment:
not sure this one is needed since you fixed the weird issue by using `sudo apt-get upgrade libstdc++6`.